tests/scripts/all.sh

```diff
@@ -924,27 +924,47 @@ component_test_default_cmake_gcc_asan () {
     tests/context-info.sh
 }
 
-component_test_full_cmake_gcc_asan () {
-    msg "build: full config, cmake, gcc, ASan"
+component_test_full_make_clang_asan () {
```

Comment:
https://ci.trustedfirmware.org/job/mbed-tls-pr-merge/job/PR-6614-merge/8/flowGraphTable/ doesn't list this job. What's going on?

The pre-test check should have caught `test_full_make_clang_asan` from `all.sh --list-all-components`, and should have either complained that none of Docker images can run it or decided to dispatch on one of the Docker images, based on `all.sh --list-components` output. And the Ubuntu 20.04 image should have included `test_full_make_clang_asan` in its `all.sh --list-components`.

The Jenkins logs don't show the ouput of `all.sh --list-…`, unfortunately. The pipeline has no branch for `all_u20-test_full_make_clang_asan`. So where did `test_full_make_clang_asan` go?

Reply:
Our Groovy code is silently skipping ubuntu-20.04 jobs :frowning: Patch incoming.

Reply:
Fix: https://github.com/Mbed-TLS/mbedtls-test/pull/76